code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -425,6 +425,7 @@
 	if(jobban_isbanned(target, ROLE_CULTIST))
 		return
 	var/mob/living/simple_animal/hostile/construct/C = new c_type(get_turf(target))
+	new /obj/effect/particle_effect/smoke/sleeping(target.loc)
```

Comment:
I don't think this should happen by default. Maybe attach an arg `creates_smoke = FALSE`. Then when clicking on Narsie, set that to TRUE: `make_new_construct(..., creates_smoke = TRUE)`

Reply:
Well, I could do that, if it's needed

Reply:
Please do so @larentoun